Directory.Build.props

```diff
@@ -9,6 +9,8 @@
     <PackageLicenseUrl>https://github.com/jet/equinox/blob/master/LICENSE</PackageLicenseUrl>
     <Copyright>Copyright © 2016-8</Copyright>
 
+    <RepoDir>$([System.IO.Path]::GetFullPath("$(MSBuildThisFileDirectory)"))</RepoDir>
```

Comment:
Can you coalesce with ThisDirAbsolute ?

Reply:
yes, but the issue is that i need it in a fsproj, the `cli\Equinox.Cli\Equinox.Cli.fsproj` like

```xml
    <Content Include="$(RepoDir)/bin/equinox-cli/net461/*">
```

if i wrote it as `$(ThisDirAbsolute)/bin/equinox-cli/net461/*` and it's confusing, because is not that file dir, but the repo dir

Reply:
Maybe remove L28-30 and use RepoDir below that ? I could argue for the duplication staying too so will leave it up to your taste - just wanted to make sure the duplication was intentional and not an oversight